basculechecks/provide.go

```diff
@@ -28,10 +27,9 @@ func ProvideMetricValidator(optional bool) fx.Option {
 // ProvideCapabilitiesMapValidator is an uber fx Provide() function that builds
 // a MetricValidator that uses a CapabilitiesMap and ConstChecks, using the
 // configuration found at the key provided.
-func ProvideCapabilitiesMapValidator(key string) fx.Option {
+func ProvideCapabilitiesMapValidator() fx.Option {
 	return fx.Options(
 		fx.Provide(
-			arrange.UnmarshalKey(key, CapabilitiesMapConfig{}),
 			NewCapabilitiesMap,
```

Comment:
For my clarification - with this, since we are no longer unmarshaling the capabilities config (and others) in bascule, we're planning to move the unmarshaling to the server which is calling ProvideCapabilitiesMapValidator?

Reply:
Yes, that's right.